Tweak wording to avoid use of gender-specific language (#91417)
Azure/azure-content: articles/active-directory/manage-apps/grant-consent-single-user.md

In this article, you'll learn how to grant consent on behalf of a single user by using PowerShell.

> When a user grants their own consent, the following events occur:

Reviewer: Let's go with the following:

```suggestion
When a user grants consent for themselves, the following events occur
```
